contrib/quickstart/kratos/oidc/kratos.yml

```diff
@@ -0,0 +1,106 @@
+version: v0.13.0
+
+dsn: memory
+
+serve:
+  public:
+    base_url: http://localhost:4433/
+    cors:
+      enabled: true
+  admin:
+    base_url: http://kratos:4434/
+
+session:
+  whoami:
+    required_aal: aal1
+
+selfservice:
+  default_browser_return_url: http://localhost:4455/
+  allowed_return_urls:
+    - http://localhost:4455
+    - http://localhost:19006/Callback
+    - exp://localhost:8081/--/Callback
+
+  methods:
+    password:
+      enabled: true
+    oidc:
+      enabled: true
+      config:
+        providers:
+          - id: linkedin
+            label: LinkedIn
+            provider: linkedin_v2
+            client_id: 78s2hvsam6tyun
+            client_secret: qe2TYfjWYFsVfFaI
```

Comment:
Do you want to keep those? :sweat_smile:

Reply:
Those were just test credentials. I removed them now.